devdbsetup

```diff
@@ -105,7 +106,7 @@ do
 done
 PASS="$(security find-generic-password -a "$(whoami)" -s "$PASS_KEY" -w)"
 
-createRemoteSchemaDump
+createRemoteSchemaDump || true
```

Comment:
hide that || true bad boy inside the function instead.

Reply:
done